io.catenax.week_based_capacity_group/3.0.1/WeekBasedCapacityGroup.ttl

```diff
@@ -0,0 +1,447 @@
+   samm:characteristic :LinkedCapacityGroupsCharacteristic ;
+   samm:exampleValue "be4d8470-2de6-43d2-b5f8-2e5d3eebf3fd" .
+
+:unitOfMeasureIsOmitted a samm:Property ;
+   samm:preferredName "Unit Of Measure Is Omitted"@en ;
+   samm:description "Explicit indicator of whether the unit of measure is left out of the payload intentionally. If \"true\" it means the sending application sends demand values without unit of measure intentionally and the unit of measure MUST NOT be contained in the payload. If \"false\" a unit of measure MUST be supplied."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue false .
+
+:demandVolatilityParameters a samm:Property ;
+   samm:preferredName "Demand Volatility Parameters"@en ;
+   samm:description "The parameters to calibrate the measuring of demand volatility."@en ;
+   samm:characteristic :DemandVolatilityCharacteristic .
+
+:capacityGroupIsInactive a samm:Property ;
+   samm:preferredName "Capacity Group Is Inactive"@en ;
+   samm:description "Indicates that this capacity group is currently not in use/maintained by the supplier."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:CapacityGroupName a samm:Characteristic ;
+   samm:preferredName "Capacity Group Name"@en ;
+   samm:description "The name of a capacity group, used to distinguish it from other capacity groups."@en ;
+   samm:dataType xsd:string .
+
+:SupplierLocationsCharacteristic a samm-c:Set ;
+   samm:preferredName "Supplier Locations Characteristic"@en ;
+   samm:description "Contains a set of locations the supplier fulfills a demand request from."@en ;
+   samm:dataType xsd:string ;
+   samm-c:elementCharacteristic :BPNSTrait .
+
+:LinkedDemandSeriesSet a samm-c:Set ;
+   samm:preferredName "Linked Demand Series Set"@en ;
+   samm:description "A set of demand series assigned to a certain capacity group."@en ;
+   samm:dataType :LinkedDemandSeries .
+
+:CapacityTimeSeries a samm-c:SortedSet ;
+   samm:preferredName "Capacity Time Series"@en ;
+   samm:description "The weekly actual and maximum capacities in a given time period."@en ;
+   samm:dataType :Capacity .
+
+:Timestamp a samm:Characteristic ;
+   samm:preferredName "Timestamp"@en ;
+   samm:description "Point in time."@en ;
+   samm:dataType xsd:dateTimeStamp .
+
+:LinkedCapacityGroupsCharacteristic a samm-c:Set ;
+   samm:preferredName "Linked Capacity Groups Characteristic"@en ;
+   samm:description "Contains all capacity groups which are linked to the current one."@en ;
+   samm:dataType xsd:string ;
+   samm-c:elementCharacteristic :UUIDv4IdTrait .
+
+:DemandVolatilityCharacteristic a samm:Characteristic ;
+   samm:preferredName "Demand volatility parameters characteristic"@en ;
+   samm:description "The characteristics of the demand volatility parameterization."@en ;
+   samm:dataType :DemandVolatilityParametersEntity .
+
+:BPNSTrait a samm-c:Trait ;
```

Comment:
You can directly make use of BpnsTrait from shared model instead of declaring your own trait. https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/io.catenax.shared.business_partner_number/2.0.0/BusinessPartnerNumber.ttl#L73

Reply:
Since all of our aspect models (4 in total) currently utilize BPNL/BPNS this way, we do not intend to change it for a single model, but would rather change it all at once. Also this would need to be formally discussed and decided within the expert group. 
Conclusion: We are not spontaneously going to do this now.

Reply:
> Since all of our aspect models (4 in total) currently utilize BPNL/BPNS this way, we do not intend to change it for a single model, but would rather change it all at once. Also this would need to be formally discussed and decided within the expert group. Conclusion: We are not spontaneously going to do this now.

I agree with @ClosedSourcerer. As this is PR is just a patch, nothing should be changed which could impact the compatibility.